code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -264,10 +264,77 @@
 	dispensable_reagents = list("water", "ice", "milk", "soymilk", "coffee", "tea", "hot_coco", "cola", "spacemountainwind", "dr_gibb", "space_up",
 	"tonic", "sodawater", "lemon_lime", "grapejuice", "sugar", "orangejuice", "lemonjuice", "limejuice", "tomatojuice", "banana",
 	"watermelonjuice", "carrotjuice", "potato", "berryjuice")
+	/var/list/special_reagents2 = list(list(""),
```

Comment:
`/var/l`

Not how you start a var.

Reply:
Same as above.

Reply:
it's the improper way of doing it. You should be relying on the style guide.

Copy-pasting and relying on bad former code is not an excuse.

Reply:
What style guide?

Reply:
https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md

Reply:
Thanks.

Reply:
Just to ensure I'm on the right track here do I do;

`	/var/special_reagents2 = list(list(""),
							list("bananahonk", "milkshake", "cafe_latte", "cafe_mocha"),
							list("triple_citrus", "icecoffe","icetea"))`

Or;

`	/var/special_reagents2 = list()
	special_reagents2 = list(list(""),
							list("bananahonk", "milkshake", "cafe_latte", "cafe_mocha"),
							list("triple_citrus", "icecoffe","icetea"))`

Given there's no full example it's just a bit confusing to me ^-^'

Reply:
Derped trying to post code...screenshotting instead.

Just to ensure I'm on the right track.

Which of these is right?

![code1](https://user-images.githubusercontent.com/12178527/52759252-cf5a1080-2fd0-11e9-9d4b-45da10def2c6.png)

![code2](https://user-images.githubusercontent.com/12178527/52759262-d2ed9780-2fd0-11e9-8911-0a1b879854a8.png)

The style guide isn't too easy too follow for me, not worked on something like this PR before, so I'm unsure.

Reply:
Pretend that first line has another list behind (list(""), 

As that was my intention x.x